src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -371,6 +400,17 @@ export default class TaskRoleContainerList extends React.Component {
     }
   }
 
+  showDialog(logListUrl) {
+    const { hideDialog } = this.state;
+
+    getContainerLogList(logListUrl).then(({ fullLogUrls, _ }) => {
+      this.setState({
+        hideDialog: !hideDialog,
+        dialogMessage: fullLogUrls.locations,
```

Comment:
This state, `dialogMessage`, is redundant.

Use `fullLogUrls.locations` is enough.